client/src/Components/coffeeShop/CoffeeShopForm.js

```diff
@@ -23,7 +23,7 @@ const CoffeeShopForm = ({
     menu: "",
     website: "",
     contact_info: "",
-    cost: 0,
+    cost: 3, //TODO this is always what it will work
```

Comment:
can you explain the change about this too.

Reply:
Yeah that is just if you create a new shop it will only be set to that number. Edit same thing, can't change it. Not sure where the issue happened.